### PR TITLE
Update provisioningStatus field name for assignment

### PR DESCRIPTION
**Pull request by abhinav-ghai**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by abhinav-ghai** [comment]
Breaking change is intended. This is not yet released to any customer.

**Review by ChenTanyi — APPROVED** [review]

**Comment by ChenTanyi** [comment]
@abhinav-ghai Could you ask a code owner or someone in your team to review it?

File: specification/automanage/resource-manager/Microsoft.Automanage/preview/2020-06-30-preview/examples/createOrUpdateConfigurationProfileAssignment.json [hunk 1]
```diff
@@ -38,13 +38,13 @@
...
           "targetId": "/subscriptions/subscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Compute/virtualMachines",
-          "configurationProfile": "Azure Best Practices - Prod",
+          "configurationProfile": "Azure virtual machine best practices – Production",
           "accountId": "/subscriptions/subid/resourceGroups/rg/providers/Microsoft.Automanage/AutomanageAccounts/AutomanageAccount",
           "configurationProfilePreferenceId": "/subscriptions/subscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Automanage/configurationProfilePreferences/defaultProfilePreference",
           "compliance": {
             "updateStatus": "Succeeded"
           },
-          "provisioningStatus": "Succeeded"
+          "provisioningState": "Succeeded"
```
Comment (stowen-msft) [inline specification/automanage/resource-manager/Microsoft.Automanage/preview/2020-06-30-preview/examples/createOrUpdateConfigurationProfileAssignment.json:47]: this was the core change needed

**Review by stowen-msft — APPROVED** [review]

**Comment by abhinav-ghai** [comment]
> @abhinav-ghai Could you ask a code owner or someone in your team to review it?

Done, reviewed by Stephen Owen from Automanage team

**Comment by ChenTanyi** [comment]
@abhinav-ghai please fix the merge conflict.

**Pull request merged**